features/home/src/main/kotlin/com/teacherworkout/features/home/HomeFeature.kt

```diff
@@ -7,9 +7,15 @@ import androidx.navigation.navigation
 import com.teacherworkout.commons.ui.navigation.AppDestinations
 
 fun NavGraphBuilder.homeFeature(_navHostController: NavHostController) {
-    navigation(startDestination = AppDestinations.Home.landing, route = AppDestinations.Features.home) {
+    navigation(
+        startDestination = AppDestinations.Home.landing,
+        route = AppDestinations.Features.home
+    ) {
         composable(route = AppDestinations.Home.landing) {
             LandingScreen()
         }
+        composable(route = AppDestinations.Home.profile) {
```

Comment:
My proposal is like this

App references: Account, Home
Home references: Profile, Learn

So when the app starts it can determine if the user is logged and then set the start route to either Account of Home.

Home will manage it's own start route logic, right now it default to Landing (Acasa)

Reply:
Yeap, that is another good way to handle our navigation